src/store/model/evga.ts

```diff
@@ -15,108 +15,246 @@ export const Evga: Store = {
-    },
-    {
-      brand: 'evga',
-      model: 'ftw3',
-      series: '3060ti',
-      url: 'https://www.evga.com/products/product.aspx?pn=08G-P5-3665-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'ftw3 ultra',
-      series: '3060ti',
-      url: 'https://www.evga.com/products/product.aspx?pn=08G-P5-3667-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'xc3 black',
-      series: '3070',
-      url: 'https://www.evga.com/products/product.aspx?pn=08G-P5-3751-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'xc3 ultra',
-      series: '3070',
-      url: 'https://www.evga.com/products/product.aspx?pn=08G-P5-3755-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'ftw3 ultra',
-      series: '3070',
-      url: 'https://www.evga.com/products/product.aspx?pn=08G-P5-3767-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'xc3 black',
-      series: '3080',
-      url: 'https://www.evga.com/products/product.aspx?pn=10G-P5-3881-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'ftw3 ultra',
-      series: '3080',
-      url: 'https://www.evga.com/products/product.aspx?pn=10G-P5-3897-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'ftw3 ultra hydro copper',
-      series: '3080',
-      url: 'https://www.evga.com/products/product.aspx?pn=10G-P5-3897-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'ftw3',
-      series: '3080',
-      url: 'https://www.evga.com/products/product.aspx?pn=10G-P5-3895-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'xc3',
-      series: '3080',
-      url: 'https://www.evga.com/products/product.aspx?pn=10G-P5-3883-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'xc3 ultra',
-      series: '3080',
-      url: 'https://www.evga.com/products/product.aspx?pn=10G-P5-3885-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'xc3 black',
-      series: '3090',
-      url: 'https://www.evga.com/products/product.aspx?pn=24G-P5-3971-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'ftw3 ultra',
-      series: '3090',
-      url: 'https://www.evga.com/products/product.aspx?pn=24G-P5-3987-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'ftw3',
-      series: '3090',
-      url: 'https://www.evga.com/products/product.aspx?pn=24G-P5-3985-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'xc3',
-      series: '3090',
-      url: 'https://www.evga.com/products/product.aspx?pn=24G-P5-3973-KR',
-    },
-    {
-      brand: 'evga',
-      model: 'xc3 ultra',
-      series: '3090',
-      url: 'https://www.evga.com/products/product.aspx?pn=24G-P5-3975-KR',
-    },
+/**   {
+  brand : 'evga',
+  labels : {
+    inStock: {
+         container: '#EVGAContent > div > div.main-wrapper > div.main-product',
+         text: ['100-BR-0450-K1'],
+             },
+            },
+         model: 'test:model',
+         series: 'test:series',
+          url: 'https://www.evga.com/products/feature.aspx',
+ },**/
```

Comment:
Do you mind removing this if it's not in use?

Reply:
Sure